src/network/rpc/CMakeLists.txt

```diff
@@ -1,35 +1,20 @@
 # Note: run make rpc_stubs to re-generate rpc classes
-# TODO: refactor this cmake
 
-cmake_minimum_required(VERSION 3.16)
-set(CPP_NAMESPACE taraxa::net)
-
-include(../../../submodules/taraxa-aleth/cmake/EthDependencies.cmake)
-include(../../../submodules/taraxa-aleth/cmake/EthExecutableHelper.cmake)
+include(${CMAKE_SOURCE_DIR}/submodules/taraxa-aleth/cmake/EthDependencies.cmake)
+include(${CMAKE_SOURCE_DIR}/submodules/taraxa-aleth/cmake/EthExecutableHelper.cmake)
 
 find_program(ETH_JSON_RPC_STUB jsonrpcstub)
-add_custom_target(gen_rpc_stubs)
-add_custom_target(rpc_stubs)
 
-foreach (api_name Net Taraxa Test)
+set(CPP_NAMESPACE taraxa::net)
+file(GLOB API_DEF_FILES "${CMAKE_CURRENT_SOURCE_DIR}/*.jsonrpc.json")
+add_custom_target(rpc_stubs)
+foreach (api_def_file ${API_DEF_FILES})
+    get_filename_component(api_name ${api_def_file} NAME_WE)
     jsonrpcstub_create(
-            gen_rpc_stubs ${api_name}.jsonrpc.json
+            rpc_stubs ${api_name}.jsonrpc.json
```

Comment:
well gen_rpc_stubs is definitely more clear in case this custom target is still supposed to generate rpc stubs :)

Reply:
I will rename it to gen_rpc_stubs

Reply:
done